src/TabBar.tsx

```diff
@@ -472,7 +472,7 @@ const styles = StyleSheet.create({
     flex: 1,
   },
   scroll: {
-    overflow: 'scroll',
+    overflow: 'hidden',
```

Comment:
Hey! Thanks for the PR, but this breaks React Native.
```suggestion
    overflow: Platform.select({ default: 'scroll', web: undefined }),
```